mitosheet/css/toolbar.css

```diff
@@ -200,13 +219,59 @@
 }
 
 .mito-toolbar-tabbar {
-  margin-bottom: 5px;
   display: flex;
+  justify-content: space-between;
+}
+
+.mito-toolbar-tabbar-right, .mito-toolbar-tabbar-left {
+  display: flex;
+  align-content: center;
+  flex-direction: row;
+  height: 26px;
 }
 
 .mito-toolbar-tabbar-selected-underline {
   width: 100%;
   height: 2px;
   background: var(--mito-highlight);
   border-radius: 2px;
+}
+
+.mito-toolbar-button-dropdown-icon {
+  display: flex;
+  margin-left: 5px;
+  font-size: 7px;
+  align-self: center;
+}
+
+.mito-toolbar-number-precision {
+  display: flex;
+}
+
+.mito-toolbar-number-precision .select-text {
+  font-size: 11px;
+}
+
+.mito-toolbar-container .select-container {
+  display: flex;
+  flex-direction: row;
+  height: 22px;
+  box-sizing: border-box;
+  width: 150px;
+}
```

Comment:
Can we actually modify the select component to take a `height` prop? That way, we're not modifying the CSS from an unexpected location, and the component is well contained.

Reply:
This goes for all the select css modification in the file.